### DISCUSSION TITLE
Share the same version of dependencies among workspaces

### DISCUSSION THREAD

**Discussion by alamothe** [yarnpkg/berry#3727] [Q&A]
Hello,
My workspaces have dependencies on e.g. `koa`, `koa-bodyparser`, `koa-router` etc. 

Each of them lists that dependency in `package.json`. 

However, I would like to force them to use the same version, and upgrade version together. Is there any solution for this?

**Comment by danielpza** [comment]
See https://yarnpkg.com/features/constraints

Outside of yarn there's also this tool https://github.com/JamieMason/syncpack/
